src/Lhm/Lhm.php

```diff
@@ -152,12 +154,12 @@ public static function cleanup($run = false, array $options = [])
                 $adapter->query("DROP TRIGGER IF EXISTS {$trigger}");
             }
 
+            $adapter->query('SET foreign_key_checks=0');
```

Comment:
I was too -- I'm not sure. I will look into it more, but it definitely had foreign keys.